client/src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+/**
+ * @class Footer
+ * @classdesc  returns the footer component
+ */
+export default class Footer extends React.Component {
```

Comment:
You can shorten this by using `import React, { Component } from 'react'` and you just extend Component.
That way you import the Component object too from react directly

Reply:
Oh i didn't know that. Thanks so much.